packages/data-provider/src/controllers/submittedDataController.ts

```diff
@@ -15,7 +16,7 @@ const controller = (dependencies: BaseDependencies) => {
 	const defaultPageSize = 20;
 	return {
 		deleteSubmittedDataBySystemId: async (
-			req: Request<{ systemId: string }, {}, any, { dryRun: string; comment: string }>,
+			req: Request<{ systemId: string }, object, any, { dryRun: string; comment: string }>,
```

Comment:
This is a good update. `{}` as a type just indicates "not null or undefined". `object` specifies something that is at least an object.